### PR TITLE
Added the IPFS project

### PR DESCRIPTION
**Pull request by RichardLitt**
Discussed here initially: https://github.com/ipfs/community/issues/94.

### CODE REVIEW

File: _data/projects/ipfs.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: IPFS
+desc: The InterPlanetary File System: The Permanent Web
```
Comment (shiftkey) [inline _data/projects/ipfs.yml]: If you want to keep the colon here, wrap the value in quote marks

Comment (RichardLitt) [inline _data/projects/ipfs.yml]: Done.

File: _data/projects/ipfs.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+name: IPFS
+desc: "The InterPlanetary File System: The Permanent Web"
+site: https://ipfs.io
+tags:
+# Note those are tags categorizing your project, not issue labels.
+- *javascript*
```
Comment (shiftkey) [inline _data/projects/ipfs.yml]: The YAML parser continues to complain - these entries shouldn't be wrapped in `*`s

Comment (RichardLitt) [inline _data/projects/ipfs.yml]: Fixed!

**Comment by shiftkey** [comment]
Thanks!

**Comment by RichardLitt** [comment]
Thank you!

**Pull request merged**